builder-support/dockerfiles/Dockerfile.wforce

```diff
@@ -1,5 +1,5 @@
-FROM ubuntu:bionic as wforce
-
+FROM ubuntu:focal as wforce
```

Comment:
Any reason you're using Ubuntu? For pdns/rec/dnsdist we specifically picked Debian.

Reply:
I don't remember if there was a specific reason, although this is just for building packages, so does it matter?

Reply:
Oh right, this is not the wforce Docker image. Ignore me :)